packages/plugin-webpack/plugin.js

```diff
@@ -54,28 +55,38 @@ module.exports = function plugin(config, args) {
         extendConfig = (cfg) => ({ ...cfg, ...args.extendConfig });
       }
 
-      let dom = new JSDOM(
-        fs.readFileSync(path.join(srcDirectory, config.devOptions.fallback))
```

Comment:
Is `devOptions.fallback` no longer used?

Reply:
We don't need it anymore; all html files emitted (including the fallback file) will be processed as entry points.

I wasn't quite sure what to do with [args.skipFallbackOutput](https://github.com/pikapkg/create-snowpack-app/blob/c049fbbf9e2ee1291fe4f0af7611f4583546728c/packages/plugin-webpack/plugin.js#L221) though. The name doesn't make sense anymore (it now prevents output to all html files), but removing/renaming the option would be a breaking change to the public api.

Reply:
@drwpow Have any thoughts on the options and config options?

Reply:
It’s looking like `devOptions.fallback` is used in Snowpack dev server for stuff, so that’s fine for the webpack plugin to ignore it. AFAIK the intent of that config wasn’t originally for the webpack plugin, so seems fine to remove.

As for `skipFallbackOutput`, if we don’t need it… we don’t need it? It looks like it never got documented anyway so no harm no foul if it’s removed 🤷‍♂️. There aren’t any references for that in Snowpack. While I think you’re right that that would technically be a breaking change, I don’t think it would cause any errors to remove as it would just be ignored.